### ISSUE TITLE
Conda package

### ISSUE THREAD

**Issue by thomas-mueller** [kubeflow/katib#1410]
/kind feature

In addition to the existing pip package it would be nice to publish a conda package. Such packages already exist for other parts of kubeflow, e.g. pipelines.

**Comment by iamthebot** [comment]
Reopening this. I'm taking a stab at a conda-forge feedstock today.

**Comment by iamthebot** [comment]
Thanks for the ping. @andreyvelich would you be willing to be listed as a maintainer? It only involves merging an auto-generated PR when a new version is released on pypi.

WIP PR is [here](https://github.com/conda-forge/staged-recipes/pull/25603)

**Comment by andreyvelich** [comment]
Thank you for creating this @iamthebot!
It would be nice to update Katib SDK README on how we should auto-generate PR for a new version: https://github.com/kubeflow/katib/tree/master/sdk/python/v1beta1#publish-new-sdk-version-to-pypi

>you be willing to be listed as a maintainer?

Absolutely, please also add @johnugeorge @tenzen-y.
@droctothorpe Do you want to be part of maintainer list for conda package ?

It would be great to publish Training Operator SDK to conda as well: https://github.com/kubeflow/training-operator/tree/master/sdk/python

**Comment by iamthebot** [comment]
@andreyvelich that's the neat thing, once this PR (adding kubeflow-katib) is merged we'll get a new repo in the conda-forge org. Conda-forge's build system will automatically create a new PR tagging the maintainers whenever you publish a new version in pypi. No need to trigger generation yourself.

I'll go ahead and add those folks as maintainers. **You'll need to comment on the PR I linked saying you are ok with being listed as a maintainer.**

Once this PR is ready to go I'll work on one for training operator SDK. I'll make a new issue on that repo to track the conda packaging effort.

**Comment by droctothorpe** [comment]
> Thank you for creating this @iamthebot!
> It would be nice to update Katib SDK README on how we should auto-generate PR for a new version: https://github.com/kubeflow/katib/tree/master/sdk/python/v1beta1#publish-new-sdk-version-to-pypi
> 
> >you be willing to be listed as a maintainer?
> 
> Absolutely, please also add @johnugeorge @tenzen-y.
> @droctothorpe Do you want to be part of maintainer list for conda package ?
> 
> It would be great to publish Training Operator SDK to conda as well: https://github.com/kubeflow/training-operator/tree/master/sdk/python
> 

Sure thing and +1 to publishing the training SDK to conda.

**Comment by iamthebot** [comment]
@droctothorpe @andreyvelich @johnugeorge @tenzen-y please leave a comment on [this PR](https://github.com/conda-forge/staged-recipes/pull/25603) indicating you are willing to be listed as a maintainer for the katib conda package. The conda-forge PR cannot be merged otherwise. Thanks.

> Sure thing and +1 to publishing the training SDK to conda.
Yep, will make a new PR in a few minutes for the training SDK.

**Comment by tenzen-y** [comment]
@iamthebot Thanks for your work.
/remove-good-first-issue 
/remove-help

**Comment by iamthebot** [comment]
@tenzen-y conda package [is published](https://anaconda.org/conda-forge/kubeflow-katib). Can we close this issue?

**Comment by tenzen-y** [comment]
> @tenzen-y conda package [is published](https://anaconda.org/conda-forge/kubeflow-katib). Can we close this issue?

@iamthebot Sure, thank you for this effort!
/close

**Comment by iamthebot** [comment]
@andreyvelich will do, kind of busy today but will try and get to it tomorrow.

**Comment by andreyvelich** [comment]
Absolutely, take your time @iamthebot!
Thank you for your contributions!
